.travis.yml

```diff
@@ -12,6 +12,7 @@ cache:
 script:
   - yarn test:noWatch
   - yarn build:prod
+  - yarn build:staging
```

Comment:
The staging URLs need to be pass into this command as they are not provided in this repo. I recommend we provide them via environment variables via TravisCI.

The full command should looks something like:
```bash
BUILD_ORIGIN=${STAGING_DOMAIN} API=staging API_ORIGIN=${STAGING_DOMAIN} yarn build:staging
```

Reply:
There is an extra step that needs to be done to correct source maps that are being done manually today.

```bash
sed -i 's/sourceMappingURL=OneSignal/sourceMappingURL=Staging-OneSignal/' Staging-*.js
sed -i 's/sourceMappingURL=OneSignal/sourceMappingURL=Staging-OneSignal/' Staging-*.css
```